Makefile.config

```diff
@@ -28,17 +28,14 @@ LIBS += -ldl
 endif
 
 # Compiler options
-CFLAGS       += $(CYC_PROFILING) $(CYC_GCC_OPT_FLAGS) -fPIC -Wall -Wno-shift-negative-value -Wno-unused-command-line-argument -Iinclude
-BASE_CFLAGS  ?= $(CYC_PROFILING) $(CYC_GCC_OPT_FLAGS) -fPIC -Wall -Wno-shift-negative-value -Wno-unused-command-line-argument -I$(PREFIX)/include
+CFLAGS       += $(CYC_PROFILING) $(CYC_GCC_OPT_FLAGS) -fPIC -Wall -Wno-shift-negative-value -Wno-unused-command-line-argument
+BASE_CFLAGS  ?= $(CYC_PROFILING) $(CYC_GCC_OPT_FLAGS) -fPIC -Wall -Wno-shift-negative-value -Wno-unused-command-line-argument
```

Comment:
It's unclear to me why `-Iinclude` and `-I$(PREFIX)/include` was needed here in the first place.

Reply:
At a minimum, without these the GitHub CI fails:

```
cc  -O2 -fPIC -Wall -Wno-shift-negative-value -Wno-unused-command-line-argument   -c -o scheme/base.o scheme/base.c
scheme/base.c:469:10: fatal error: cyclone/types.h: No such file or directory
  469 | #include "cyclone/types.h"
      |          ^~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [<builtin>: scheme/base.o] Error 1
Error: Process completed with exit code 2.
```

Reply:
Ah right, these options are used to compile the C code so the `-I` is actually needed. I readded it.